src/main/java/com/severell/initializr/action/template/TemplateFileOperation.java

```diff
@@ -24,7 +24,7 @@
 
     TemplateFileOperation(TemplateParameter param){
         parameter = param;
-        dirPath = System.getProperty("java.io.tmpdir").concat( this.getFileSeparator()).concat(parameter.getName());
+        dirPath = Config.get("TEMPLATE_DIR").concat( this.getFileSeparator()).concat(parameter.getName());
```

Comment:
This PR looks good, though, just an opinionated view, personally, i prefer getting the template directory ourselves instead of making a dev going through the trouble of configuring it manually.

Reply:
What do you mean by configuring it manually? As in having to have the TEMPLATE_DIR in the .env file?

Reply:
Actually, you make a good point. I was thinking of when we deploy this, we probably want to have a single place for the templates to live. In case we need to have more than one instance running. But as a dev you don't want to have to keep updating or creating that template. So we could auto generate those when running locally but when in PROD we can pull from a directory